src/attributes/attr.js

```diff
@@ -3,11 +3,14 @@ define([
 	"../core/access",
 	"./support",
 	"../var/rnotwhite",
+	"./val",
```

Comment:
We can compare it like that https://github.com/pgilad/jquery/compare/6df1bf9be818b8955f94113aecc689fdf7a2e172...ab692d396542b1a65a93460359fe1d8c6ca6ccf6

Although, i'm also not pushing for specific land strategy